src/components/game/Board.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import { GameOverStatus } from '../../hooks/useGame.tsx';
+import type { CellsType } from '../types/GameType.tsx';
+import { GameOverStatus } from '../types/GameType.tsx';
 import Cell from './Cell';
 import GameOverModal from './GameOver.tsx';
-interface BoardProps {
-  cells: (number | null)[][];
+
+type BoardProps = {
+  cells: CellsType;
   gameOver: GameOverStatus;
-}
+  checkInit: () => void;
+};
 
-const Board: React.FC<BoardProps> = ({ cells, gameOver }) => {
+const Board: React.FC<BoardProps> = ({ cells, gameOver, checkInit }) => {
```

Comment:
사소한건데 React.FC는 잘 안 씁니다!
```suggestion
const Board = ({ cells, gameOver, checkInit }: BoardProps) => {
```

Reply:
React.FC 안쓰는 이유

https://velog.io/@frombozztoang/React.FC-%EC%82%AC%EC%9A%A9-%EC%A7%80%EC%96%91%ED%95%98%EA%B8%B0